desktop/notification/notificationtest/fdo.go

```diff
@@ -0,0 +1,189 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
```

Comment:
Nitpick: 2021?

Reply:
It was 2020 when I wrote this code :smile:  I've updated it now.